### DISCUSSION TITLE
Adding openvpn split tunelling

### DISCUSSION THREAD

**Discussion by wocsom92** [RaspAP/raspap-webgui#1655] [Ideas]
It would be fantastic to have the option to configure split tunneling directly from the RaspAP GUI. Currently, when using an OpenVPN profile, the GUI does not provide a way to specify a list of domains that should be accessed via the VPN.

**Comment by billz** [comment]
@frankozland as an OpenVPN / RaspAP power user, do you feel this would be useful?
